qiskit/ignis/verification/entanglement/analysis.py

```diff
@@ -15,10 +16,17 @@
     HAS_MATPLOTLIB = False
 
 
-def ordered_list_generator(counts_dictionary, qn):
+def ordered_list_generator(counts_dictionary: Dict, qn: int) -> np.array:
     """
     For parity oscillations; just arranges dictionary of counts
     in bitwise binary order to compute dot products more easily
+
+    Args:
+            counts_dictionary: The counts
```

Comment:
This tab indent seems pretty deep especially compared to other `Args` entries in this file. It looks like there are an extra 4 spaces for these 2 params.

Reply:
Fixed, thanks